src/python/pants/engine/internals/build_files_test.py

```diff
@@ -581,6 +592,9 @@ def test_macro_undefined_symbol_bootstrap() -> None:
         rules=[QueryRule(AddressFamily, [AddressFamilyDir])],
         is_bootstrap=True,
     )
+    rule_runner.set_options(
+        args=("--build-file-prelude-globs=prelude.py",),
+    )
```

Comment:
Unrelated change. This test didn't actually test macro parsing as that content was never loaded.